adopt-a-dog/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>ADOPTADOG Home</title>
+  <link rel="stylesheet" type="text/css"
+  href="styles/styles.css">
+</head>
+<body>
+  <img src="adoptadog-logo.svg" alt="Adopt a dog logo">
+  <header>
+    <img src="images/christopher-harris-57366-unsplash.jpg" alt="Heading image">
+    <div>
+      <h1>ADOPTADOG</h1>
+      <h2>Seattle's premiere dog adoption agency</h2>
+    </div>
+    <div>
+      <nav>
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="dogs.html">Dogs</a></li>
+          <li><a href="blog.html">Blog</a></li>
+        </ul>
+      </nav>
+    </div>
+  </header>
+  <main>
+    <div>
+      <article>
+        <img src="images/eric-ward-610868-unsplash.jpg" alt="Man hugging dog">
+        <h3>
```

Comment:
you'll want to keep these all on one line like you did below. <h2>Some Text</h2>